core/shared/src/main/scala/fs2/concurrent/PubSub.scala

```diff
@@ -253,14 +253,14 @@ private[fs2] object PubSub {
           }
 
         def subscribe(selector: Selector): F[Boolean] =
-          state.modify { ps =>
+          update { ps =>
```

Comment:
not that important, but I'd rather `update` be called something else (if possible). I looked at it at first glance, and got confused by assuming if was `Ref.update`